.github/workflows/nightly.yml

```diff
@@ -18,7 +18,6 @@ jobs:
         ref: develop
     - uses: actions/setup-node@v3
       with:
-        node-version: 16
```

Comment:
I still dont want this to be removed. It is not recommended  setup-node and the nightly script has to work all the time. 
if it breaks, i dont want to stress us to investigate.

Reply:
I restored the fixed version to the latest LTS - https://endoflife.date/nodejs